src/sdc11073/mdib/containerbase.py

```diff
@@ -65,14 +65,8 @@ def _updateFromNode(self, node):
         for dummy_name, cprop in self._sortedContainerProperties():
             cprop.updateFromNode(self, node)
 
-
-    def mkCopy(self, copy_node=True):
-        copied = copy.copy(self)
-        if copy_node:
-            cpNode = copy.deepcopy(self.node)
```

Comment:
There were calls with "copy_node" being "True".

Can you explain why this is in your opinion not needed anymore nonetheless?

Also there is still at least one call with an explicit "copy_node=False" as far as I can tell.

Reply:
this in unnecessary because the node is irrelevant in all cases where `mkCopy` is used. you can even set `node=None` in the copy

Reply:
It might be true for the use cases in sdc11073 itself, but not necessarily for all other use cases of the package as well.

So, isn't this technically a breaking change? 
I think you are not supposed to introduce breaking changes into 1.1.x , whenever they are not bugfixes.

Reply:
even though it is not necessary for this package, I reverted these changes in order to prevent a breaking change.